index.html

```diff
@@ -44,8 +44,138 @@ <h6><a href="">LET'S CONNECT</a></h6>
                 </ul>
             </aside>
         </section>
-       
     </main>
+
+    1
+    <section class = "works">
+        <div class = "cards">
+                <img src = "Images/Tonic-1.png" alt = "own-portfolio-page"/>
+            <div class = "cards-description">
+                <h2 class = "cards-titles">Tonic</h2>
+                <div class = "subheading-list">
+                    <span class = "canopy">CANOPY</span>
+                    <img src = "Images/dot-symbol.svg" alt = "option-symbol"/>
+                    <span class = "subheading-list-title">Back End Dev</span>
+                    <img src = "Images/dot-symbol.svg" alt = "option-symbol" />
+                    <span class = "subheading-list-title">2015</span>
+                </div>
+                <article class = "card-content">
+                    <p>A daily selection of privately personalized reads;
+                         no accounts or sign-ups required.
+
+                    </p>
+                </article> 
+                <ul class = "web-languages">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button class = "see-project">See project</button>
+            </div>
+        </div>
+    </section>
+
+    2
+    <section class = "works">
+        <div class = "cards">
+                <img src = "Images/stories-1.png" alt = "about-story"/>
```

Comment:
- [ ] I would like to suggest using a background image for this section instead of an `img` tag. This approach would be more suitable as the image is primarily intended for design purposes.
- [ ]  By applying a background image, you can have more flexibility in positioning and styling elements within the section. 🎴